--- origsrc/xvidcore-1.1.3/build/generic/configure.in	2005-11-22 15:01:33.000000000 -0600
+++ src/xvidcore-1.1.3/build/generic/configure.in	2007-07-04 19:56:44.750000000 -0500
@@ -228,7 +228,13 @@
 	SHARED_EXTENSION="so"
 	OBJECT_EXTENSION="o"
 	;;
-     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*|mingw32*|mks*)
+     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*)
+	AC_MSG_RESULT([.dll .a .o])
+	STATIC_EXTENSION="a"
+	SHARED_EXTENSION="dll"
+	OBJECT_EXTENSION="o"
+	;;
+     mingw32*|mks*)
 	AC_MSG_RESULT([.dll .a .obj])
 	STATIC_EXTENSION="a"
 	SHARED_EXTENSION="dll"
@@ -279,7 +285,14 @@
 	SPECIFIC_LDFLAGS="-Wl,-soname,libxvidcore.\$(SHARED_EXTENSION).\$(API_MAJOR) -shared -lc -lm"
 	SPECIFIC_CFLAGS="-fPIC"
 	;;
-     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*|mingw32*|mks*)
+     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*)
+	AC_MSG_RESULT([ok])
+	STATIC_LIB="libxvidcore.\$(STATIC_EXTENSION)"
+	SHARED_LIB="cygxvidcore-\$(API_MAJOR).\$(SHARED_EXTENSION)"
+	SPECIFIC_LDFLAGS="-shared -Wl,--enable-auto-image-base,--out-implib,libxvidcore.dll.a"
+	SPECIFIC_CFLAGS=""
+	;;
+     mingw32*|mks*)
 	AC_MSG_RESULT([ok])
 	STATIC_LIB="xvidcore.\$(STATIC_EXTENSION)"
 	SHARED_LIB="xvidcore.\$(SHARED_EXTENSION)"
