diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x platform.inc -x configure origsrc/xvidcore-1.1.3/build/generic/configure.in src/xvidcore-1.1.3/build/generic/configure.in
--- origsrc/xvidcore-1.1.3/build/generic/configure.in	2005-11-22 15:01:33.000000000 -0600
+++ src/xvidcore-1.1.3/build/generic/configure.in	2007-07-04 19:56:44.750000000 -0500
@@ -228,7 +228,13 @@
 	SHARED_EXTENSION="so"
 	OBJECT_EXTENSION="o"
 	;;
-     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*|mingw32*|mks*)
+     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*)
+	AC_MSG_RESULT([.dll .a .o])
+	STATIC_EXTENSION="a"
+	SHARED_EXTENSION="dll"
+	OBJECT_EXTENSION="o"
+	;;
+     mingw32*|mks*)
 	AC_MSG_RESULT([.dll .a .obj])
 	STATIC_EXTENSION="a"
 	SHARED_EXTENSION="dll"
@@ -279,7 +285,14 @@
 	SPECIFIC_LDFLAGS="-Wl,-soname,libxvidcore.\$(SHARED_EXTENSION).\$(API_MAJOR) -shared -lc -lm"
 	SPECIFIC_CFLAGS="-fPIC"
 	;;
-     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*|mingw32*|mks*)
+     [[cC]][[yY]][[gG]][[wW]][[iI]][[nN]]*)
+	AC_MSG_RESULT([ok])
+	STATIC_LIB="libxvidcore.\$(STATIC_EXTENSION)"
+	SHARED_LIB="cygxvidcore-\$(API_MAJOR).\$(SHARED_EXTENSION)"
+	SPECIFIC_LDFLAGS="-shared -Wl,--enable-auto-image-base,--out-implib,libxvidcore.dll.a"
+	SPECIFIC_CFLAGS=""
+	;;
+     mingw32*|mks*)
 	AC_MSG_RESULT([ok])
 	STATIC_LIB="xvidcore.\$(STATIC_EXTENSION)"
 	SHARED_LIB="xvidcore.\$(SHARED_EXTENSION)"
